src/qt/receivecoinsdialog.cpp

```diff
@@ -133,7 +146,11 @@ void ReceiveCoinsDialog::on_receiveButton_clicked()
     QString address;
     QString label = ui->reqLabel->text();
     /* Generate new receiving address */
-    address = model->getAddressTableModel()->addRow(AddressTableModel::Receive, label, "");
+    OutputType address_type = g_address_type;
```

Comment:
Maybe use model->getDefaultAddressType() here instead of g_address_type to be more consistent.

Reply:
yes then perhaps next line only use `address_type` for comparison.

Reply:
Done